_posts/2021-05-22-fluxv2-openfaas.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "See how Flux's Git Ops ToolKit can simplify delivery"
+description: "See how Flux can simplify serverless deployments to OpenFaaS"
+date: 2021-05-22    
+image: /images/2021-05-22-flux/earth.jpg
+categories:
+- arkade
+- kubectl
+- gitops
+- openfaas-operator
+- flux
+- CI/CD
+author_staff_member: alistair
+dark_background: true
+
+---
+
+We learn how to manage OpenFaaS and install functions using Flux.
+
+# What is Flux
+Flux is an open source project by [weaveworks](https://www.weave.works/oss/flux/) which enables 
+application delivery using GitOps. Your cluster config is stored in Git and the flux controllers continually work to ensure
+that the cluster config matches the state in Git. This allows us to take the "cattle not pets" mentality applied to modern cloud computing further and apply the same thinking
+to our clusters. 
+
+We no longer need to manually issue `helm install` or `kubectl apply` commands to manage our clusters and 
+configuration changes are audited and easily reviewed as Git PullRequests.
+
+![flux v2 diagram](/images/2021-05-22-flux/gitops-toolkit.png)
+*image from [Flux](https://github.com/fluxcd/flux2/blob/main/docs/_files/gitops-toolkit.png)*
+ 
+# GitOps
+
+`GitOps` is a set of practices that aim to democratise our "Operations" type practices such as deploying new application 
+versions or installing new software on servers that takes practices used in development such as using Git as a `single source of truth`
+and building auditability and accountability through peer review on Pull Requests to manage our day-to-day operations activities.
+
+I won't go into much mode detail as there are entire conferences dedicated to this topic and a few paragraphs from me won't 
+do this topic justice! Take some time to familiarise yourself with the concept if it is new to you. 
+
+### Assumptions
+
+* You have a Github Account (any git server can be used, please see section on installing flux for a link to the flux docs)
```

Comment:
Add the link or remove the mention.

Having a GitHub account is pretty much assumed, so you could slip this into the introduction instead where you say what the blog post will cover.